### PR TITLE
layers: Removed duplicate dynamicOffsets from core_validation

### PR DESCRIPTION
**Pull request by tobine**
Missed this from a rebase conflict. Kill top-level cmd buffer node
dynamicOffsets vector and only use vector based off of lastBound data.

### CODE REVIEW

**Comment by lentinem** [comment]
+1

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -2866,36 +2866,36 @@ static VkBool32 validate_and_update_drawtime_descriptor_state(
...
-                                    reinterpret_cast<const uint64_t &>(set_node->set), __LINE__, DRAWSTATE_DYNAMIC_OFFSET_OVERFLOW,
-                                    "DS",
-                                    "VkDescriptorSet (%#" PRIxLEAST64 ") bound as set #%u has dynamic offset %#" PRIxLEAST32 ". "
-                                    "Combined with offset %#" PRIxLEAST64 " and range %#" PRIxLEAST64
-                                    " from its update, this oversteps its buffer "
-                                    "(%#" PRIxLEAST64 ") which has a size of %#" PRIxLEAST64 ".",
-                                    reinterpret_cast<const uint64_t &>(set_node->set), i, pCB->dynamicOffsets[dynOffsetIndex],
-                                    pWDS->pBufferInfo[j].offset, pWDS->pBufferInfo[j].range,
-                                    reinterpret_cast<const uint64_t &>(pWDS->pBufferInfo[j].buffer), bufferSize);
+                                result |= log_msg(dev_data->report_data, VK_DEBUG_REPORT_ERROR_BIT_EXT,
```
Comment (chrisforbes) [inline layers/core_validation.cpp]: This is a duplicate of the previous block...

Comment (tobine) [inline layers/core_validation.cpp]: Thanks, Chris. Force pushed an update that kills that block as well. Darn rebase conflicts.

**Comment by tobine** [comment]
cherry-picked and pushed to master

**Pull request closed without merging**